composer.json

```diff
@@ -125,8 +125,7 @@
     "git-hook-install": "php tools/contrib/install.php",
     "git-hook-uninstall": "php tools/contrib/uninstall.php",
     "cldr-init": "PrestaShop\\PrestaShop\\Core\\Cldr\\Composer\\Hook::init",
-    "phpunit-sf": "php bin/phpunit -c admin-dev/",
-    "phpdoc-sf": "php vendor/phpdocumentor/phpdocumentor/bin/phpdoc.php -d src -t docs/phpdoc-sf --template=\"responsive-twig\""
```

Comment:
This costs ~ 2M of data, for no gain. In months I've never see someone in core team use it.